src/firmware/app/control/robot_controller.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "firmware/app/control/trajectory_planner.h"
+#include "firmware_new/boards/frankie_v1/io/drivetrain.h"
+#include "firmware_new/boards/frankie_v1/io/drivetrain_unit.h"
```

Comment:
This file is in the `app` folder but mixing `app/io` includes as well as `app/io` functions. They should be separated.